config/default/manager_webhook_patch.yaml

```diff
@@ -20,4 +20,4 @@ spec:
       - name: cert
         secret:
           defaultMode: 420
-          secretName: opendatahub-operator-controller-webhook-cert
+          secretName: opendatahub-operator-webhook-service
```

Comment:
So why is this change?  I mean
1) why to rename?
2) I see service.beta.openshift.io/serving-cert-secret-name: opendatahub-operator-webhook-cert in Service and would expect the same here to mount as a volume but looks like there is some other logic behind it now. In the bundle it is different. I see in the message "- secret: default application namespace + other default ones + istio cert" Could you share some link?

Excuse my ignorance :)

Reply:
i am trying to remember the reason why it was changed.
will need to dig into the old commit and find what exactly happen, but was something related to the bundle csv and the way it creates service for webhook which does not match the one if we test locally for e2e.